{"variants":[{"paths":["\/documentation\/vigramsdk\/bluetoothserviceerror\/notconnected(_:)"],"traits":[{"interfaceLanguage":"swift"}]}],"kind":"symbol","primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"notConnected"},{"kind":"text","text":"("},{"kind":"typeIdentifier","preciseIdentifier":"s:10Foundation4UUIDV","text":"UUID"},{"kind":"text","text":")"}],"platforms":["iOS"]}]}],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/BluetoothServiceError\/notConnected(_:)"},"metadata":{"title":"BluetoothServiceError.notConnected(_:)","symbolKind":"case","modules":[{"name":"VigramSDK"}],"externalID":"s:9VigramSDK21BluetoothServiceErrorO12notConnectedyAC10Foundation4UUIDVcACmF","role":"symbol","roleHeading":"Case","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"notConnected"},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"UUID","preciseIdentifier":"s:10Foundation4UUIDV"},{"kind":"text","text":")"}]},"schemaVersion":{"patch":0,"major":0,"minor":3},"hierarchy":{"paths":[["doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK","doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/BluetoothServiceError"]]},"sections":[],"abstract":[{"text":"Connecting to a device failed unexplicably.","type":"text"}],"references":{"doc://com.vigram.VigramSDK/documentation/VigramSDK":{"kind":"symbol","type":"topic","identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK","abstract":[],"title":"VigramSDK","url":"\/documentation\/vigramsdk","role":"collection","images":[{"type":"icon","identifier":"logo.png"}]},"doc://com.vigram.VigramSDK/documentation/VigramSDK/BluetoothServiceError/notConnected(_:)":{"role":"symbol","type":"topic","url":"\/documentation\/vigramsdk\/bluetoothserviceerror\/notconnected(_:)","title":"BluetoothServiceError.notConnected(_:)","kind":"symbol","identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/BluetoothServiceError\/notConnected(_:)","abstract":[{"text":"Connecting to a device failed unexplicably.","type":"text"}],"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"notConnected"},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"UUID","preciseIdentifier":"s:10Foundation4UUIDV"},{"kind":"text","text":")"}]},"doc://com.vigram.VigramSDK/documentation/VigramSDK/BluetoothService":{"title":"BluetoothService","abstract":[{"type":"text","text":"A BluetoothService is used to handle bluetooth connections."}],"role":"symbol","url":"\/documentation\/vigramsdk\/bluetoothservice","fragments":[{"text":"protocol","kind":"keyword"},{"text":" ","kind":"text"},{"text":"BluetoothService","kind":"identifier"}],"navigatorTitle":[{"text":"BluetoothService","kind":"identifier"}],"type":"topic","kind":"symbol","identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/BluetoothService"},"doc://com.vigram.VigramSDK/documentation/VigramSDK/BluetoothServiceError":{"url":"\/documentation\/vigramsdk\/bluetoothserviceerror","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"BluetoothServiceError","kind":"identifier"}],"role":"symbol","identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/BluetoothServiceError","abstract":[{"type":"text","text":"BluetoothServiceError lists different errors that can be propagated by "},{"type":"reference","identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/BluetoothService","isActive":true},{"type":"text","text":" implementations."}],"type":"topic","title":"BluetoothServiceError","kind":"symbol","navigatorTitle":[{"text":"BluetoothServiceError","kind":"identifier"}]},"logo.png":{"alt":"Vigram logo.","variants":[{"url":"\/images\/com.vigram.VigramSDK\/logo.png","traits":["1x","light"]}],"type":"image","identifier":"logo.png"}}}